ignite/pkg/cosmosanalysis/module/module.go

```diff
@@ -22,57 +22,58 @@ type Msgs map[string][]string
 // Module keeps metadata about a Cosmos SDK module.
 type Module struct {
 	// Name of the module.
-	Name string
+	Name string `json:"name,omitempty"`
 
 	// GoModulePath of the app where the module is defined.
-	GoModulePath string
+	GoModulePath string `json:"go_module_path,omitempty"`
 
 	// Pkg holds the proto package info.
-	Pkg protoanalysis.Package
+	Pkg protoanalysis.Package `json:"package,omitempty"`
 
 	// Msg is a list of sdk.Msg implementation of the module.
-	Msgs []Msg
+	Msgs []Msg `json:"messages,omitempty"`
 
 	// HTTPQueries is a list of module queries.
-	HTTPQueries []HTTPQuery
+	HTTPQueries []HTTPQuery `json:"http_queries,omitempty"`
 
 	// Types is a list of proto types that might be used by module.
-	Types []Type
+	Types []Type `json:"types,omitempty"`
 }
 
 // Msg keeps metadata about an sdk.Msg implementation.
 type Msg struct {
 	// Name of the type.
-	Name string
+	Name string `json:"name,omitempty"`
 
 	// URI of the type.
-	URI string
+	URI string `json:"uri,omitempty"`
 
-	// FilePath is the path of the .proto file where message is defined at.
-	FilePath string
+	// File path is the path of the proto file where message is defined.
```

Comment:
Go's documentation standard:

```suggestion
	// FilePath is the path of the proto file where message is defined.
```